apps/web/src/pages/brand/tabs/v2/BrandingForm.tsx

```diff
@@ -0,0 +1,232 @@
+                          minWidth: '6.25rem',
+                          display: 'flex',
+                        })}
+                      >
+                        {!field.value ? (
+                          <Stack gap="4px" direction="column" align="center" justify="center" flexGrow={1}>
+                            <ActionButton onClick={() => dropzoneRef.current?.()} Icon={IconOutlineFileUpload} />
+                            <Text color={colors.B70}>Upload</Text>
+                          </Stack>
+                        ) : (
+                          <img
+                            data-test-id="logo-image-wrapper"
+                            src={field.value}
+                            className={css({
+                              minHeight: '6.25rem',
+                              minWidth: '6.25rem',
+                              objectFit: 'contain',
+                            })}
+                            alt="avatar"
+                          />
+                        )}
+                      </div>
+                    </Dropzone>
+                  </div>
+                )}
+                control={control}
+                name="brandLogoUrl"
+              />
+            </Stack>
+
+            <div style={{ width: '15rem' }}>
+              <Controller
+                render={({ field }) => (
+                  <ColorInput label="Brand color" data-test-id="brand-color-picker" disallowInput={false} {...field} />
+                )}
+                control={control}
+                name="brandColorValue"
+              />
+            </div>
+
+            <div style={{ width: '15rem' }}>
+              <Controller
+                render={({ field }) => (
+                  <Select
+                    label="In-App font family"
+                    placeholder="Select a font family"
+                    data={[
+                      'inherit',
+                      'Fira Code',
+                      'Roboto',
+                      'Montserrat',
+                      'Open Sans',
+                      'Lato',
+                      'Nunito',
+                      'Oswald',
+                      'Raleway',
+                    ]}
+                    data-test-id="font-family-selector"
+                    {...field}
+                  />
+                )}
+                control={control}
+                name="fontFamilyValue"
+              />
+            </div>
+
+            <div style={{ width: '15rem' }}>
+              <Controller
+                render={({ field }) => (
+                  <ColorInput
+                    label="In-App font color"
+                    data-test-id="font-color-picker"
+                    disallowInput={false}
+                    {...field}
+                  />
+                )}
+                control={control}
+                name="fontColorValue"
+              />
+            </div>
+          </Stack>
```

Comment:
Interestingly, the font color value was send as color (brand color) in the previous iteration.

Reply:
😱